java/configuration/src/main/java/sleeper/configuration/properties/instance/CommonProperty.java

```diff
@@ -194,6 +194,19 @@ public interface CommonProperty {
             .validationPredicate(Utils::isTrueOrFalse)
             .propertyGroup(InstancePropertyGroup.COMMON)
             .runCdkDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty TRANSACTION_LOG_SNAPSHOT_CREATION_BATCH_SIZE = Index.propertyBuilder("sleeper.metadata.transactionlog.snapshot.creation.batch.size")
+            .description("The number of tables to create transaction log snapshots for in a single invocation. A separate invocation " +
+                    "of the lambda will be made for each batch when there are more tables than the batch size.")
+            .defaultValue("5")
```

Comment:
This description needs updating with the FIFO queue, and it should probably default to 1.